### PR TITLE
MacOS specific fix for problems with image acquisition.

### PR DESCRIPTION
**Pull request by DSep**
MacOS specific fix for problems with image acquisition running within the ground station code.  Bless Andrei for this.
Originally we were ensuring the mapbox/map is initialized only once, but we don't need that.

### CODE REVIEW

File: main_window/mapping/mapbox_utils.py [hunk 1]
```diff
@@ -262,8 +270,10 @@ def getRow(row):
                     i.getImage(overwrite=overwrite)
                     a = a - 1
 
-        with concurrent.futures.ThreadPoolExecutor() as executor:
-            executor.map(getRow, self.ta)
+        for row in self.ta:
+            getRow(row)
+        # with concurrent.futures.ThreadPoolExecutor() as executor:
+        #     executor.map(getRow, self.ta)
```
Comment (SilentSys) [inline main_window/mapping/mapbox_utils.py:276]: This needs to be undone i guess

Comment (ADSteele916) [reply main_window/mapping/mapbox_utils.py:276]: My higher-order functions will not be taken away.

**Review by ADSteele916 — DISMISSED** [review]
Looks good to me. The tests seem to be working properly again.

**Review by SilentSys — DISMISSED** [review]
LGTM. Merge conflict though.

**Review by SilentSys — APPROVED** [review]

**Review by ADSteele916 — APPROVED** [review]
LGTM

**Pull request merged**